src/views/products/ProductForm.vue

```diff
@@ -22,6 +22,26 @@
         <InputSupplier v-model="product.supplier_id" />
       </div>
 
+      <!-- Unit Price -->
+      <div class="p-field">
+        <label>Unit Price</label>
```

Comment:
just "Price" is better I think, otherwise it can be confusing regarding the unit (piece, kg..)